opentera-webrtc-native-client/OpenteraWebrtcNativeClient/CMakeLists.txt

```diff
@@ -84,3 +84,8 @@ add_subdirectory(python)
 if (OPENTERA_WEBRTC_ENABLE_TESTS)
     add_subdirectory(test)
 endif ()
+
```

Comment:
Currently, it is checked in the CMakeLists of the subdirectory, because the file that instantiates either a GStreamer decoder/encoder or a webrtc builtin decoder/encoder based on this settings in inside the GStreamer component.
I already added a TODO to move this logic out of the component, so that it is possible to build completely without it.